lib/SILOptimizer/Transforms/AccessEnforcementOpts.cpp

```diff
@@ -142,13 +142,13 @@ namespace {
 /// Reachability results are stored here because very few accesses are
 /// typically in-progress at a particular program point,
 /// particularly at block boundaries.
-using DenseAccessMap = llvm::SmallMapVector<unsigned, BeginAccessInst *, 4>;
```

Comment:
I think it would make more sense to call this `DenseAccessSet` (the vector part is an implementation detail of the Set).

Reply:
fixed in new PR